packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -51,6 +51,7 @@ import {
 	getStorageIds,
 	summarizeBlobManagerState,
 	toRedirectTable,
+	// eslint-disable-next-line import/no-deprecated
```

Comment:
Will it help you to find/remove these if you put a keyword on these lines?  Or... maybe you can just do `git revert` on this commit to start the removal branch?

Reply:
`npm run lint:fix` can remove them